hera_mc/cm_active.py

```diff
@@ -174,6 +176,49 @@ def load_info(self, at_date=None):
             self.info.setdefault(key, [])
             self.info[key].append(info)
 
+    def load_rosetta(self, at_date=None):
+        """
+        Retrieve the current 'rosetta' mappings.
+
+        Since the mappings are all "current" there is no date.  They also don't
```

Comment:
This docstring needs to be fixed -- there is a date.

Reply:
I also am adding an "add_part_rosetta" function for scripting